common/lib/xmodule/xmodule/public/js/vertical_student_view.js

```diff
@@ -0,0 +1,16 @@
+/* JavaScript for Vertical Student View. */
+window.VerticalStudentView = function (runtime, element) {
+
+    RequireJS.require(['js/bookmarks/views/bookmark_button'], function (BookmarkButton) {
```

Comment:
@symbolist Thanks. That was my understanding too, so it is good to hear that it works as expected in the sandbox environment. Would you be able to update the wiki to add this explanation. Thanks!